### PR TITLE
Add flag to specify custom configuration file

### PR DESCRIPTION
**Pull request by leourbina**
This allows creating several different configuration files within the same project, to allow for switching between environment configurations such as local vs qa. 

@timmfin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,3 +78,12 @@ default:
...
   port: 8888
 ```
+
+Multiple Configurations
+-----------------------
+
+You may want to have multiple configuration files within the same project, in order to allow different proxying rules depending on the envirnoment you are working on (e.g. local vs QA). You can specify a custom config file by using the `--config` flag as follows:
+
+```bash
+sudo vee --config .vee.qa
```
Comment (b-ash) [inline README.md]: It's probably not necessary to imply that sudo is needed for vee. That's only if your port is a reserved port.

**Comment by b-ash** [comment]
I haven't had a need for this yet, but I can totally see the benefit. I dig it. :boat:

**Comment by timmfin** [comment]
:+1: 

Could you bump the version on this to `1.1.0` too?

**Comment by leourbina** [comment]
Version bumped. Unless there are any other comments, I'll merge this some time tomorrow afternoon.

**Comment by timmfin** [comment]
I've tagged the commit and published v1.1.0 to npm. Thanks again!

**Pull request merged**